app/src/main/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -44,6 +43,7 @@
 
   private final PermissionManager permissionManager;
   private final PermissionService permissionService;
+  private final String INTERSTITIAL_NETWORK_MOPUB = "MoPub";
```

Comment:
Is there any advantage of extracting this here? Couldn't this be inside each method call (impression/click) in the AppViewManager?

Reply:
fixed